db/models.py

```diff
@@ -25,6 +30,11 @@ class Movie(models.Model):
     def __str__(self) -> str:
         return self.title
 
+    class Meta:
+        indexes = [
+            models.Index(fields=["title"])
+        ]
+
```

Comment:
The `class Meta` should be defined after all attributes of the `Movie` class. Currently, it is correctly placed after the `__str__` method, which is acceptable. However, ensure that all attributes are defined before `class Meta`.